yaksh/models.py

```diff
@@ -296,7 +296,7 @@ def _create_lesson_copy(self, user):
         lesson_files = self.get_files()
         new_lesson = self
         new_lesson.id = None
-        new_lesson.name = "Copy of {0}".format(self.name)
+        new_lesson.name = self.name
```

Comment:
If the name will be same as original lesson name then no need of assigning the same name.